nautobot/dcim/homepage.py

```diff
@@ -87,6 +87,14 @@ def _connected_powerports_count(request):
                 description="Represents a set of devices which share a common control plane",
                 weight=400,
             ),
+            HomePageItem(
+                name="Redundancy Group",
+                link="dcim:redundancygroup_list",
+                model=models.RedundancyGroup,
+                permissions=["dcim.view_redundancygroup"],
+                description="Represents a set of devices which operate in a failover/HA group",
+                weight=400,
```

Comment:
weight should be unique, possibly 450? Or else 500 and increment all the items below accordingly.